app/Policies/TaskPolicy.php

```
@@ -46,7 +46,7 @@ public function create(User $user): bool
      */
     public function update(User $user, Task $task): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$task'. (undefined)

(UnusedFormalParameter)